tests/jest-wrapper-factory.ts

```diff
@@ -11,13 +13,19 @@ import { getVuexStore } from '@/store'
 // Utils
 import { createLocalVue, mount, shallowMount } from '@vue/test-utils'
 import { IncorporationResources } from '@/resources'
+// This polyfill(intersection-observer) was installed as a dev dependency as without it components being tested
+// that use the IntersectionObserver API results in warnings that browser does not support this API.
+// e.g. IntersectionObserver API is not available in your browser.
+//      Please install this polyfill: https://github.com/w3c/IntersectionObserver/tree/master/polyfill
+require('intersection-observer')
```

Comment:
Thanks for the comment. (It answers the question "why is this here?")

Why use "require"? Does "import" work as well?

Reply:
i think i was just looking at the documentation and just used what was provided.  Will try importing like we do normally.

Reply:
updated to use `import` from `require`